Diagnose-PrinterDriverBlock.ps1

```
@@ -0,0 +1,246 @@
+# SuperOps captures stdout, so everything goes to the console via Write-Log.
+# A transcript file is also written for central collection / later retrieval.
+$logDir = 'C:\Temp'
+if (-not (Test-Path -LiteralPath $logDir)) {
+    New-Item -Path $logDir -ItemType Directory -Force | Out-Null
+}
+$transcript = Join-Path $logDir "PrinterDriverBlock-Diag-$env:COMPUTERNAME.log"
+
+function Write-Log {
+    param([string]$Message)
+    Write-Host $Message
+    try { Add-Content -LiteralPath $transcript -Value $Message -Encoding UTF8 } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)